api/bibliotheca.py

```diff
@@ -390,7 +391,7 @@ def update_availability(self, licensepool):
         )
         return monitor.process_items([licensepool.identifier])
 
-    def _patron_activity_request(self, patron):
+        _activity_request(self, patron):
```

Comment:
This bit broke the tests.

Reply:
What the heck? I only added the timeout line. Maybe I had something highlighted and erased it. Adding it back in.